api/src/main/java/org/openmrs/module/ugandaemrreports/reporting/calculation/smc/FollowUpCalculation.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.ugandaemrreports.reporting.calculation.smc;
+
+import org.openmrs.Encounter;
+import org.openmrs.EncounterType;
+import org.openmrs.api.context.Context;
+import org.openmrs.calculation.patient.PatientCalculationContext;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.calculation.result.ListResult;
+import org.openmrs.calculation.result.SimpleResult;
+import org.openmrs.module.ugandaemrreports.reporting.calculation.AbstractPatientCalculation;
+import org.openmrs.module.ugandaemrreports.reporting.calculation.Calculations;
+import org.openmrs.module.ugandaemrreports.reporting.calculation.EmrCalculationUtils;
+import org.openmrs.module.ugandaemrreports.reporting.utils.CalculationUtils;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Collection;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by Nicholas Ingosi on 7/17/17.
+ */
+public class FollowUpCalculation extends AbstractPatientCalculation {
```

Comment:
@ningosi What in the Followup is being calculated here? It is not clear from the class name